### PR TITLE
Feature/wdp190703 14/rwd new furniture

### PR DESCRIPTION
**Pull request by DzesikaTkaczyk**

### CODE REVIEW

File: src/partials/50_section-products.html [hunk 1]
```diff
@@ -106,7 +106,7 @@ <h5>Aenean Ru Bristique</h5>
           </div>
         </div>
       </div>
-      <div class="col-3">
+      <div class="col-xl-3  col-lg-4 col-md-6 col-sm-6 col-12">
```
Comment (matwjablonski) [inline src/partials/50_section-products.html]: double space

**Comment by matwjablonski** [comment]
please resolve conflicts

**Comment by danielmark0116** [comment]
All the items inside DIV.row must be wrapped in a parent DIV.new-furniture-list[category="$value"]

Also, there should be 5 lists, each with the category set according to the TAB BTNs. I can resolve it if you all agree :)

**Comment by matwjablonski** [comment]
please resolve conflicts

File: src/partials/50_section-products.html [hunk 2]
```diff
@@ -81,110 +75,190 @@ <h5>Chair</h5>
...
-        <div class="col-3">
-          <div class="product-box">
-            <div class="photo">
-              <div class="sale">sale</div>
-              <div class="buttons">
-                <a href="#" class="btn-main-small">Quick View</a>
-                <a href="#" class="btn-main-small"
-                  ><i class="fa fa-shopping-basket"></i> ADD TO CART</a
-                >
+      <div class="new-furniture-list" data-category="chair">
```
Comment (danielmark0116) [inline src/partials/50_section-products.html]: Just before this line, there's a closing tab </div> missing, the rest is superb! :D

(First DIV.new-furniture-list with category=bed is not closed)

**Comment by matwjablonski** [comment]
please resolve conflicts

**Comment by DzesikaTkaczyk** [comment]
Rozwiązałam konflikt i przy okazji poprawiłam przyciski na zdjęciach bo zauważyłam że na masterze teraz nie są widoczne

**Pull request merged**